dist/playkit-cast-receiver.js

```diff
@@ -1,2 +1,2 @@
key:"_onAdProgress",value:function(){if(this._ad){var e=this._playerManager.getBreakClipDurationSec(),t=this._playerManager.getBreakClipCurrentTimeSec(),a=t/e;!this._timePercentEvent.AD_REACHED_25_PERCENT&&a>=.25&&(this._timePercentEvent.AD_REACHED_25_PERCENT=!0,this._sendEventAndCustomMessage(this._player.Event.AD_FIRST_QUARTILE)),!this._timePercentEvent.AD_REACHED_50_PERCENT&&a>=.5&&(this._timePercentEvent.AD_REACHED_50_PERCENT=!0,this._sendEventAndCustomMessage(this._player.Event.AD_MIDPOINT)),!this._timePercentEvent.AD_REACHED_75_PERCENT&&a>=.75&&(this._timePercentEvent.AD_REACHED_75_PERCENT=!0,this._sendEventAndCustomMessage(this._player.Event.AD_THIRD_QUARTILE)),!this._adCanSkipTriggered&&this._ad&&this._ad.skippable&&t>=this._ad.skipOffset&&(this._sendEventAndCustomMessage(this._player.Event.AD_CAN_SKIP),this._adCanSkipTriggered=!0),this._sendEventAndCustomMessage(this._player.Event.AD_PROGRESS,{adProgress:{currentTime:t,duration:e}})}}},{key:"_onMuteChange",value:function(){this._player.muted&&this._sendEventAndCustomMessage(this._player.Event.AD_MUTED)}},{key:"_onVolumeChange",value:function(){this._sendEventAndCustomMessage(this._player.Event.AD_VOLUME_CHANGED)}},{key:"_toggleAdBreakListeners",value:function(e){var t=this;e?(Object.keys(this._adTrackingEventHandlers).forEach(function(e){return t._playerManager.addEventListener(e,t._adTrackingEventHandlers[e])}),Object.keys(this._playerEventHandlers).forEach(function(e){return t._player.addEventListener(e,t._playerEventHandlers[e])}),this._adProgressIntervalId=setInterval(this._onAdProgress.bind(this),300)):(Object.keys(this._adTrackingEventHandlers).forEach(function(e){return t._playerManager.removeEventListener(e,t._adTrackingEventHandlers[e])}),Object.keys(this._playerEventHandlers).forEach(function(e){return t._player.removeEventListener(e,t._playerEventHandlers[e])}),this._adProgressIntervalId&&(clearInterval(this._adProgressIntervalId),this._adProgressIntervalId=null))}},{key:"_getAdBreakOptions",value:function(e){var t={},a=this._playerManager.getBreakManager().getBreakById(e.breakId);return a&&(t.position=a.position,t.type=this._getAdBreakTypeByPosition(a.position),t.numAds=a.breakClipIds.length),t}},{key:"_getAdBreakTypeByPosition",value:function(e){switch(e){case 0:return d.PRE;case-1:return d.POST;default:return d.MID}}},{key:"_getAdOptions",value:function(e){var t={},a=this._playerManager.getBreakManager().getBreakById(e.breakId);if(a){var n=this._playerManager.getBreakManager().getBreakClipById(e.breakClipId);t.url=n.contentId,t.contentType=n.contentType,t.title=n.title,t.position=a.breakClipIds.indexOf(n.id)+1,t.duration=n.duration,t.clickThroughUrl=n.clickThroughUrl,t.posterUrl=n.posterUrl,t.skipOffset=n.whenSkippable,t.linear=!0}return t}},{key:"_sendEventAndCustomMessage",value:function(e,t){this._logger.debug(e.toUpperCase(),t),this._player.dispatchEvent(new g(e,t)),this._context.sendCustomMessage(o.CUSTOM_CHANNEL,void 0,new y(e,t))}}]),e}();t.ReceiverAdsManager=v}])});
```

Comment:
remove prettier/eslint rules from minified and source map

Reply:
done